com.unity.multiplayer.mlapi/Runtime/Core/NetworkObject.cs

```diff
@@ -52,27 +52,29 @@ public ulong OwnerClientId
         {
             get
             {
-                if (_ownerClientId == null)
+                if (OwnerClientIdInternal == null)
```

Comment:
How come private / internal _ownerClientId is not m_ownerClientId?

Reply:
because it's `internal ulong? OwnerClientIdInternal = null;` and we're following `public` modifier logic for `internal` fields as well.